ui/options_plugins.ui

```diff
@@ -98,7 +98,7 @@
             </sizepolicy>
            </property>
            <property name="text">
-            <string>Download plugins</string>
+            <string>Reload available plugins</string>
```

Comment:
The only software I have around that offers a similar function is IntelliJ, which uses "Reload List of Plugins", which seems fine to me, as does "Reload/Refresh [the] plugin list"